manager/ytsync.go

```diff
@@ -966,3 +981,23 @@ func waitForDaemonProcess(timeout time.Duration) error {
 	}
 	return errors.Err("timeout reached")
 }
+
+func test() {
```

Comment:
should be in its own test file

Reply:
Yeah, this is me testing stuff in travis. It needs to be removed when travis passes.